misc/package/Data Files/MWSE/core/lib/dependencyManagement/types/archives.lua

```diff
@@ -0,0 +1,37 @@
+local util = require("dependencyManagement.util")
+local function isArchiveActive(archive)
+    local loadedArchives = tes3.getArchiveList()
+    for _, loadedArchive in ipairs(loadedArchives) do
+        mwse.log("archive: " .. loadedArchive)
```

Comment:
Do we want to log the archive names? If I am following correctly, this will print all the loaded archives to the mwse.log file for every mod with a metadata file with archive dependencies. So, I guess this should be removed.